lib/Chat/Command/Executor.php

```diff
@@ -132,9 +132,9 @@ protected function execHelp(Room $room, IComment $message, string $arguments): s
 						$tempHelp = substr($tempHelp, 0, strpos($tempHelp, "\n"));
 					}
 				}
-				$helps[] = $tempHelp;
+				$helps[] = $command->getCommand() .' -> ' . $tempHelp;
```

Comment:
Not sure about this. Currently all the samples print `/<sample>` already:
https://github.com/nextcloud/spreed/blob/master/sample-commands/hackernews.php#L36
```
/wiki - A simple command to find wikipedia articles for a term
/hackernews - A simple command to list the Top 5 top, new or best stories
An error occurred while running the command. Please ask an administrator to check the logs.
```

So with your change it is duplicated:
```
wiki -> /wiki - A simple command to find wikipedia articles for a term
hackernews -> /hackernews - A simple command to list the Top 5 top, new or best stories
```